Tiltfile.dev

```diff
@@ -0,0 +1,18 @@
+local_resource(
+  name = 'dev:chips-rest-interfaces-consumer',
+  cmd = 'mvn compile',
+  deps = ['src']
+)
+
+custom_build(
+  ref = '169942020521.dkr.ecr.eu-west-1.amazonaws.com/local/chips-rest-interfaces-consumer',
```

Comment:
you'll also need to ensure this image is made available in the ecr by raising a pr in aws_common_infrastructure
https://github.com/companieshouse/aws-common-infrastructure-terraform/pull/143/files